_posts/2022-05-04-#24584.md

```diff
@@ -10,15 +10,23 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+* One technique used for determining the payment address and amount is the ["Payment to different script type"](https://en.bitcoin.it/wiki/Privacy#Sending_to_a_different_script_type) heuristic.
+
+* [PR #23789](https://github.com/bitcoin/bitcoin/pull/23789) added payment address matching when generating a change address as a means of breaking the heuristic. This logic can lead to mixed UTXOs in the wallet. Depending on how these UTXOs are spent in the future, they might still leak information about which is the change/payment address in the original transaction.
+
+* This PR adds logic to avoid mixing different type UTXOs when funding a transaction
```

Comment:
Maybe explain what you mean by "different type" and "mixed" UTXOs - reminder that this should be designed for someone who doesn't have a ton of context about coin selection or wallet privacy.